### DISCUSSION TITLE
How to store user data in prisma when using sign in with apple

### DISCUSSION THREAD

**Discussion by Itsnotaka** [nextauthjs/next-auth#5280] [Help]
### Question 💬

The way how the sign was done is alright, but I noticed that there isn't a name record when creating an account using sign-in with apple. And the name record would work when using Google or GitHub sign-in.
https://next-auth.js.org/providers/apple#configuration

### How to reproduce ☕️

Simply go with the doc with an adapter to Prisma
```
adapter: PrismaAdapter(prisma),
Apple({
      clientId: env.APPLE_CLIENT_ID,
      clientSecret: env.APPLE_CLIENT_SECRET,
    }),
```

### Contributing 🙌🏽

No, I am afraid I cannot help regarding this

**Comment by nmandal** [comment]
Any update here?
